README.md

```diff
@@ -0,0 +1,45 @@
+# Deepgram Rust SDK
+
+[![CI](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml/badge.svg)](https://github.com/deepgram-devs/deepgram-rust-sdk/actions/workflows/ci.yaml)
+[![crates.io](https://img.shields.io/crates/v/deepgram)](https://crates.io/crates/deepgram)
+[![dowloads](https://img.shields.io/crates/d/deepgram)](https://crates.io/crates/deepgram)
+[![docs](https://img.shields.io/docsrs/deepgram)](https://docs.rs/deepgram)
+[![license](https://img.shields.io/crates/l/deepgram)](./LICENSE)
+
+**NOTICE**: *This SDK is still very much work-in-progress. The interface is subject to rapid change.
+If you would like a stable way to access all that Deepgram has to offer, please use the
+[Deepgram API](https://developers.deepgram.com/api-reference/) directly until this SDK has been stabilized.*
+
+Official Rust SDK for Deepgram's automated speech recognition APIs.
+
+To access the API you will need a Deepgram account. Sign up for free at [console.deepgram.com](https://console.deepgram.com/signup?utm_medium=github&utm_source=DEVREL&utm_content=deepgram-rust-sdk).
+
+You can learn more about the full Deepgram API at [developers.deepgram.com](https://developers.deepgram.com).
+
+## Install
+
+Add the following line to the `dependencies` table in your `Cargo.toml` file:
+
+```
+deepgram = "0.1.0"
+```
+
+## Development and Contributing
+
+Interested in contributing? We ❤️ pull requests!
+
+To make sure our community is safe for all, be sure to review and agree to our
+[Code of Conduct](./CODE_OF_CONDUCT.md). Then see the
+[Contribution](./CONTRIBUTING.md) guidelines for more information.
+
+## Getting Help
+
+We love to hear from you so if you have questions, comments or find a bug in the
+project, let us know! You can either:
+
+- [Open an issue](https://github.com/deepgram/[reponame]/issues/new) on this repository
```

Comment:
Was this copy-pasted from somewhere? I don't think we want `[reponame]` in there. [The link](https://github.com/deepgram/%5Breponame%5D/issues/new) doesn't seem to work through any GitHub magic.
```suggestion
- [Open an issue](https://github.com/deepgram-devs/deepgram-rust-sdk/issues/new) on this repository
```